haystack/document_store/sql.py

```diff
@@ -341,9 +347,23 @@ def write_labels(self, labels, index=None):
                 model_id=label.model_id,
                 index=index,
             )
-            self.session.add(label_orm)
+            if label.id in ids_exist_in_db:
```

Comment:
Is this what you meant to write here? The `if` and and the `else` cases do the same thing

Reply:
Oh yeah, It's a bug. I need to replace else part with add.